.github/workflows/test_integration.yml

```diff
@@ -12,3 +12,4 @@ jobs:
         username: ${{ secrets.DOCKER_USERNAME }}
         password: ${{ secrets.DOCKER_PASSWORD }}
         snapshot: true
+        tag_semver: true
```

Comment:
@elgohr Perhaps this is an even better approach? That way it will push to all relevant versions and then I can delete the `release-docker-hub.yml` file as I realized the `push` event applies to release creation as well.

Reply:
At the moment releases aren't semver and I don't plan to do so.
But we could use https://github.com/elgohr/Publish-Docker-Github-Action#tag_names

Reply:
Ah, alright! I thought the patch version might be optional. I'll move the setting over to here and remove the other file. Thanks for the fast feedback!